experiments/ClimaEarth/run_amip.jl

```diff
@@ -237,7 +242,13 @@ Note that land-sea area fraction is different to the land-sea mask, which is a b
 (masks are used internally by the coupler to indicate passive cells that are not populated by a given component model).
 =#
 
-land_area_fraction = SpaceVaryingInput(land_mask_data, "LSMASK", boundary_space)
+# Preprocess the file to be 1s and 0s before remapping into onto the grid
+land_area_fraction = SpaceVaryingInput(
+    land_mask_data,
+    "z",
+    boundary_space,
+    file_reader_kwargs = (; preprocess_func = (data) -> data >= 0),
```

Comment:
What does this preprocess_func do? Does it just select the data values >=0 ?

Reply:
The dataset contains elevation, so a the land mask is where the elevation is >=0. The preprocessing function converts the dataset into a boolean mask

This is not completely true for places inland that are below sea levels, so we still need to work on this before merging the PR.